packages/@react-spectrum/label/src/ReadOnlyField.tsx

```diff
@@ -74,7 +92,27 @@ function ReadOnlyField(props: SpectrumFieldProps, ref: RefObject<HTMLTextAreaEle
             )
         } />
       </FocusRing> 
-    </div>
+    </div>);
+
+  if (!label) {
+    return React.cloneElement(textfield, mergeProps(textfield.props, {
+      ref,
+      className
+    }));
+  }
```

Comment:
Reminder, try to replace with
```suggestion
  if (!label) {
    return textfield;
  }
```